Common/Securities/Future/FuturesExpiryFunctions.cs

```diff
@@ -95,6 +95,27 @@ public class FuturesExpiryFunctions
                     return thirdFriday.Add(new TimeSpan(13,30,0));
                 })
             },
+            // Russell2000EMini (RTY): https://www.cmegroup.com/trading/equity-index/us-index/e-mini-russell-2000_contract_specifications.html
+            {Futures.Indices.Russell2000EMini, (time =>
+                {
+                    // Trading can occur up to 9:30 a.m. Eastern Time (ET) on the 3rd Friday of the contract month
+                    var thirdFriday = FuturesExpiryUtilityFunctions.ThirdFriday(time);
+                    return thirdFriday.Add(new TimeSpan (13,30,0));
```

Comment:
What happens if the third Friday is a holiday?

Reply:
I pushed a change that changes ThirdFriday method behavior. If it is a holiday, it will return the previous business day (unit test was added for Good Friday case).